scripts/publish.ts

```diff
@@ -65,6 +65,12 @@ async function main(dryRun = false) {
     `pnpm publish --no-git-checks --tag ${npmTag}`,
     dryRun,
   )
+
+  // Print out version for workflow dispatch if npmTag is latest
+  if (npmTag === 'latest') {
+    console.log(`Printing version for workflow dispatch: ${npmTag}`)
+    console.log(`::set-output name=new_prisma_version::${npmTag}`)
```

Comment:
Please add a comment here what this does - I assume this puts it somehow into GH Actions workflow context?

Reply:
Yep, it exactly does that. I added the comment